pkg/kubelet/server/server_test.go

```diff
@@ -50,6 +50,7 @@ import (
 	"k8s.io/client-go/tools/remotecommand"
 	utiltesting "k8s.io/client-go/util/testing"
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
Thanks for the explanation. Though the root cause is buried quite deep, i think it is worth mentioning the purpose is to decdoe query parameter correctly.